js/indicia.datacomponents/jquery.idc.filterSummary.js

```diff
@@ -163,23 +172,24 @@
       // Standard params.
       if ($('#standard-params').length > 0) {
         $.each($('#standard-params'), function eachStatusFilter() {
-
-          html = addHtml(html, '<div id="filterSummary-standard-params-header"></div>');
-          html = addHtml(html, '<div id="filterSummary-standard-params"></div>');
-          functionCalls.push({
-            function: function() {
-              setTimeout(function() {
-                var title=$('#active-filter-label').text()
-                if (title) {
-                  $('#filterSummary-standard-params-header').html('<b>Standard filter (' + title + '):</b>') 
-                } else {
-                  $('#filterSummary-standard-params-header').html('<b>Standard filter:</b>') 
-                }
-              }, 100);
-              indiciaFns.setFilterSummary();
-            },
-            arg: null
-          })
+          if (!$(this).hasClass('survey-filter')) {
```

Comment:
As #standard-params is an ID, surely there will only be 1 element with this selector? So is there a need to do $.each? I'm also unsure why the survey-filter select control might have the id standard-params so need to be filtered out of this loop.?

Reply:
Agreed - I've removed the loop. Thanks for spotting that I was filtering for `survey-filter` class in wrong place. The `[surveyFilter]` control is now classed both as `es-filter-param` - to utilise the existing code in `icd.core.js` - _and_ `filter-summary` so that the `[filterSummary]` control gives a meaningful summary based on the title of the selected survey rather than the ID which is what the summary for `es-filter-param` would do.